standard_test.go

```diff
@@ -48,7 +48,7 @@ func TestCreateInitialBoardState(t *testing.T) {
 		{9, 8, []string{"one"}, 1, nil},
 		{2, 2, []string{"one", "two"}, 2, nil},
 		{2, 2, []string{"one", "two"}, 2, nil},
-		{1, 1, []string{"one", "two"}, 2, errors.New("not enough space to place snake")},
+		{1, 2, []string{"one", "two"}, 2, errors.New("not enough space to place snake")},
```

Comment:
@sjbcastro can you add this as another test case and leave the original test intact? It's still a valid test case.

Reply:
True, done